sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/api/Azure.ResourceManager.ResourceHealth.netstandard2.0.cs

```diff
@@ -40,6 +42,8 @@ public partial class ResourceHealthEventData : Azure.ResourceManager.Models.Reso
         public System.DateTimeOffset? LastUpdateOn { get { throw null; } }
         public Azure.ResourceManager.ResourceHealth.Models.ResourceHealthEventInsightLevelValue? Level { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.ResourceHealth.Models.ResourceHealthEventLink> Links { get { throw null; } }
+        public string MaintenanceId { get { throw null; } }
```

Comment:
What's the format of this id? `arm-id`, `guid` or just a string?

Reply:
this seems to be defined as a just a string in the released swagger:
[2023-10-01-preview](https://github.com/Azure/azure-rest-api-specs/blob/31b50995d26083df994011833adcaa816214b651/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-10-01-preview/ResourceHealth.json#L1371)

also in the previous [2023-07-01-preview(](https://github.com/Azure/azure-rest-api-specs/blob/31b50995d26083df994011833adcaa816214b651/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-10-01-preview/ResourceHealth.json#L1371)  (sdk was not released) 

the engineer responsible for this change is oof but i will talk to them about adding format UUID since it does seem like a guid from the response:
"maintenanceId": "bc452ddf-255b-4e8c-bdb4-818b4a9eb87a",